src/utils/Encryption/Encryption.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  encryptSymmetricAsStr,
+  decryptSymmetricStr,
+  CryptoInput,
+  EncryptedSymmetricString,
+} from '@kiltprotocol/sdk-js/build/crypto'
+
+export function encryption(
+  message: string,
+  secret: CryptoInput
+): EncryptedSymmetricString {
+  return encryptSymmetricAsStr(message, secret)
+}
+
+export function decryption(data: string, secret: CryptoInput): string | null {
+  return decryptSymmetricStr(JSON.parse(data), secret)
+}
```

Comment:
This whole file seems unnecessary. Do you need it for future work?

Reply:
I would say for future work it seems necessary. I removed the encryption modal already but kept the `Encryption.ts`.